applications/imaging_ai_segmentator/README.md

```diff
@@ -0,0 +1,214 @@
+        ├── 1.3.6.1.4.1.14519.5.2.1.7085.2626.nii
+        └── 1.3.6.1.4.1.14519.5.2.1.7085.2626_seg.nii
+
+2 directories, 3 files
+```
+
+**_Note_**
+It takes quite a few minutes when this command is run the first time.
+
+### Run the Application in Dev Environment
+
+It is strongly recommended a Python virtual environment is used for running the application in dev environment.
+
+This application only has Python implementation depending on a set of Python packages from [Pypi](https://pypi.org), however, a `build_and_install` step is needed to automate organizing Python code and downloading the model.
+
+
+Set up the Holohub environment, if not already done
+```bash
+./run setup
+```
+
+Set the environment variables for the application
+```bash
+source applications/imaging_ai_segmentator/env_settings.sh
+```
+
+Download images of a CT series from [TCIA](https://nbia.cancerimagingarchive.net/nbia-search/)
+(The following example shows downloading from the Holohub artifactory)
+
+```bash
+wget --no-parent -r -l 1  "https://urm.nvidia.com/artifactory/sw-holoscan-generic/test_data/dicom/TCIA_CT_ABDOMEN/"
+rm -f -r $HOLOSCAN_INPUT_PATH
+mkdir -p $HOLOSCAN_INPUT_PATH
+cp -r urm.nvidia.com/artifactory/sw-holoscan-generic/test_data/dicom/TCIA_CT_ABDOMEN/ $HOLOSCAN_INPUT_PATH
+```
+
+Optionally download the AI model from [MONAI Model Zoo](https://github.com/Project-MONAI/model-zoo/tree/dev/models/wholeBody_ct_segmentation), or wait till the build step to have it downloaded automatically
+
+```bash
+mkdir -p $HOLOSCAN_MODEL_PATH
+pip install gdown
+python -m gdown https://drive.google.com/uc?id=1PHpFWboimEXmMSe2vBra6T8SaCMC2SHT -O $HOLOSCAN_MODEL_PATH/model.pt
+```
+
+Install Python packages required by the application
+```bash
+pip install -r applications/imaging_ai_segmentator/requirements.txt
+```
+
+Build and install the application
+```bash
+./dev_container build_and_install imaging_ai_segmentator
+```
+
+Run the application
+```bash
+rm -f -r $HOLOSCAN_OUTPUT_PATH
+python install/imaging_ai_segmentator/app.py
+```
+
+**_Note_**
+If desired, run the application with explicitly input, output, and/or model folder path, for example
+```bash
+rm -f -r ./output
+python install/imaging_ai_segmentator/app.py -m $HOLOSCAN_MODEL_PATH -i $HOLOSCAN_INPUT_PATH -o ./output
+```
+
+Check output
+```bash
+ls $HOLOSCAN_OUTPUT_PATH
+```
+
+There should be a DICOM segmentation file with randomized file name. There should also a `saved_images_folder` containing folder named after the input DICOM series' instance UID, which in turn contains the input and segmentation image files in NIfTI format, e.g.
+```bash
+applications/imaging_ai_segmentator/output
+├── 1.2.826.0.1.3680043.10.511.3.64271669147396658491950188504278234.dcm
+└── saved_images_folder
+    └── 1.3.6.1.4.1.14519.5.2.1.7085.2626
+        ├── 1.3.6.1.4.1.14519.5.2.1.7085.2626.nii
+        └── 1.3.6.1.4.1.14519.5.2.1.7085.2626_seg.nii
+```
+
+### Run the Aapplication in Dev Container
```

Comment:
```suggestion
### Run the Application in Dev Container
```

Reply:
Wow, thanks for catching that; codespell did not catch it.